src/main/java/openblocks/common/entity/EntityHangGlider.java

```diff
@@ -190,7 +190,7 @@ public void onUpdate() {
 			}
 		} else {
 			if (varioControl.isValid()) {
-				updateVario(0); // well, our vertical velocity is zero, right?
```

Comment:
Yeah, but that way your frequency starts dropping from neutral every time you restart. Now it just waits for enough samples (== 4 ticks) until it starts.